src/Microsoft.Framework.Logging/Utils/TestLoggerFactory.cs

```diff
@@ -22,5 +22,7 @@ public ILogger Create(string name)
         public void AddProvider(ILoggerProvider provider)
         {
         }
+
+        LogLevel ILoggerFactory.MinimumLevel { get; set; } = LogLevel.Verbose;
```

Comment:
This one will change too? BTW I think @davidfowl and you are confusing implicit/explicit. An *explicit* interface implementation is when you include the interface name in the implementation (hence it's explicit). But we normally do *implicit* interface implementations, where the interface name is omitted.